insights/specs/__init__.py

```diff
@@ -98,6 +98,7 @@ class Specs(SpecSet):
     crypto_policies_bind = RegistryPoint()
     crt = RegistryPoint()
     current_clocksource = RegistryPoint()
+    cups_ppd = RegistryPoint(multi_output=True)
```

Comment:
@wushiqinlou - would you please move this spec ahead of the `current_clocksource`, that's to sort them in alphabet order.

Reply:
Updated